apps/user_ldap/lib/Jobs/UpdateGroups.php

```diff
@@ -72,27 +72,24 @@ public function __construct(
 		$this->dbc = $dbc;
 	}
 
-	/**
-	 * @return int
-	 */
-	private function getRefreshInterval() {
-		//defaults to every hour
-		return \OC::$server->getConfig()->getAppValue('user_ldap', 'bgjRefreshInterval', 3600);
-	}
-
 	/**
 	 * @param mixed $argument
+	 * @throws Exception
 	 */
 	public function run($argument) {
```

Comment:
```suggestion
	public function run($argument): void {
```

Should work since implementation can specify a more specific return type https://www.php.net/manual/en/language.oop5.variance.php